docs/389ds/FAQ/update-resolution-for-single-valued-attributes.md

```diff
@@ -231,20 +231,20 @@ DISCUSSION TOPIC: Do we need to handle pending values at all ? Assume a mod of a
 Concurrent state changes
 ========================
 
-There are many operations which lead to state changes in a SVA, they can be applied in any combination and order and on a wide range of masters.
+There are many operations which lead to state changes in a SVA, they can be applied in any combination and order and on a wide range of suppliers.
 
-To determine the correct behaviour all combinations in a two master topology are considered. If more masters are involved the variety increases, but in URP there will always be only the resolution of a current state with a state change received via replication.
+To determine the correct behaviour all combinations in a two supplier topology are considered. If more suppliers are involved the variety increases, but in URP there will always be only the resolution of a current state with a state change received via replication.
```

Comment:
two suppliers

Reply:
I think it okay as it means "a topology of two suppliers". So it should be two supplier topology.

But I think even better option will be to say: "two-supplier topology". Hence, I'll fix like that.